src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+import { getPeople } from '../api/people';
+import { PeopleTable } from '../components/PeopleTable';
+
+export const People = () => {
+  const [loading, setLoading] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  useEffect(() => {
+    setLoading(true);
+
+    setTimeout(() => {
+      getPeople()
+        .then(setPeople)
+        .catch(() => setErrorMessage('Something went wrong'))
+        .finally(() => setLoading(false));
+    }, 300);
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {loading && <Loader />}
+
+          {errorMessage && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              {errorMessage}
+            </p>
+          )}
+
+          {people.length === 0 && (
+            <p data-cy="noPeopleMessage">There are no people on the server</p>
+          )}
```

Comment:
The condition `people.length === 0` for displaying the 'no people' message should also check if `!loading` to avoid showing this message while data is still being fetched.